### ISSUE TITLE
I don't use passivbot,I can use it?

### ISSUE THREAD

**Issue by goushijie** [ecoppen/futuresboard#116]
**Describe the solution you'd like**
I don't use passivbot, but I still want to use it, OK?
**Describe alternatives you've considered**
I don't use passivbot, but I still want to use it, OK?

**Comment by ecoppen** [comment]
Yes! There's no requirement to use passivbot at all, this dashboard links directly with the exchange to read data from there. 

The project simply came to life from my own experience with passivbot and so I link back to it as a thanks - I will update the docs to make this clearer.
